odoh_server.go

```diff
@@ -80,6 +81,7 @@ func main() {
 	endpoints := make(map[string]string)
 	endpoints["Target"] = queryEndpoint
 	endpoints["Health"] = healthEndpoint
+	endpoints["pk"] = pkEndpoint
```

Comment:
Let's call this "PublicKey", and `publicKeyEndpoint` to match?

```suggestion
	endpoints["PublicKey"] = publicKeyEndpoint
```